docs/root/start/sandboxes/route-mirror.rst

```diff
@@ -0,0 +1,146 @@
+    route-mirror-service1-1          "/usr/local/bin/star…"   service1            running (healthy)
+    route-mirror-service1-mirror-1   "/usr/local/bin/star…"   service1-mirror     running (healthy)
+    route-mirror-service2-1          "/usr/local/bin/star…"   service2            running (healthy)
+    route-mirror-service2-mirror-1   "/usr/local/bin/star…"   service2-mirror     running (healthy)
+
+Step 2: Demonstrate static mirror cluster name
+**********************************************
+
+.. code-block:: console
+
+  $ pwd
+  envoy/examples/route-mirror
+  $ curl localhost:8080/service/1
+  Hello from behind Envoy (service 1)!
+
+The command above sends a request to the ``envoy-front-proxy`` service which forwards the request to
+``service1`` and also sends the request to the service 1 mirror, ``service1-mirror``.
+
+Step 3: See Logs
+****************
+
+.. code-block:: console
+
+   $ docker-compose logs service1
+
+   ...
+   Host: localhost:8080
+   192.168.80.6 - - [06/Oct/2022 03:56:22] "GET /service/1 HTTP/1.1" 200 -
+
+   $ docker-compose logs service1-mirror
+
+   ...
+   Host: localhost-shadow:8080
+   192.168.80.6 - - [06/Oct/2022 03:56:22] "GET /service/1 HTTP/1.1" 200 -
+
+The above logs from the ``service1`` and ``service1-mirror`` services show that
+both the ``service1`` and ``service1-mirror`` services got the request.
+
+You can also see that for the request to the ``service1-mirror`` service, the
+``Host`` header was modified by Envoy to have a ``-shadow`` suffix in the
+hostname.
+
+Step 4: Demonstrate mirror cluster via header
+*********************************************
+
+In this step, we will see a demonstration where the request specifies via a header, ``x-mirror-cluster``,
+the cluster that envoy will mirror the request to.
+
+.. code-block:: console
+
+  $ pwd
+  envoy/examples/route-mirror
+  $ curl --header "x-mirror-cluster: service2-mirror" localhost:8080/service/2
+  Hello from behind Envoy (service 2)!
+
+The command above sends a request to the ``envoy-front-proxy`` service which forwards the request to
+``service2`` and also mirrors the request to the cluster named, ``service2-mirror``.
+
+
+Step 4: See Logs
+****************
+
+.. code-block:: console
+
+   $ docker-compose logs service2
+
+   ...
+   Host: localhost:8080
+   192.168.80.6 - - [06/Oct/2022 03:56:22] "GET /service/2 HTTP/1.1" 200 -
+
+   $ docker-compose logs service2-mirror
+
+   ...
+   Host: localhost-shadow:8080
+   192.168.80.6 - - [06/Oct/2022 03:56:22] "GET /service/2 HTTP/1.1" 200 -
+
+The above logs show that both the ``service2`` and ``service2-mirror`` services
+got the request.
+
+You can also see that for the request to the ``service2-mirror`` service, the
+``Host`` header was modified by Envoy to have a ``-shadow`` suffix in the
+hostname.
+
+If you do not specify the ``x-mirror-cluster`` in the request to ``service2``,
+``service2-mirror`` will not receive the request.
+
```

Comment:
could you add a `seealso` section with links to any other reference docs and api please